test/lua_libs/util.lua

```diff
@@ -215,6 +215,23 @@ local function map_bucket_protection(test_run, cluster, value)
         [[vshard.storage.internal.is_bucket_protected = ...]], value)
 end
 
+local function router_cfg(cfg, router)
+    local vcfg = require('vshard.cfg')
```

Comment:
`reload_evolution/storage` fails if `vshard.cfg` is included not inside the function:

```
[009] reload_evolution/storage.test.lua
[009]
[009] [Instance "storage_2_a" returns with non-zero exit code: 1]
[009]
[009] Last 15 lines of Tarantool Log file [Instance "storage_2_a"][/home/serpentian/Programming/tnt/vshard/test/var/009_reload_evolution/storage_2_a.log]:
[009] Starting instance storage_2_a...
[009] started
[009] Start failed: ...rd/test/var/vshard_git_tree_copy/vshard/storage/init.lua:2045: attempt to call field 'split' (a nil value)
[009] [ fail ]

```

Reply:
Have you investigated why? Could be a bug in storage code.

Reply:
I didn't, the error looks strange, have no ideas on the reason. Created a ticket for this, https://github.com/tarantool/vshard/issues/452. Will try to investigate it later, don't have time for it right now(